diff --git a/system.post_update.php b/system.post_update.php
index 8e82b19..2c3d4f6 100644
--- a/system.post_update.php
+++ b/system.post_update.php
@@ -125,6 +125,7 @@ function system_post_update_remove_asset_query_string() {
  * Update description for view modes.
  */
 function system_post_update_add_description_to_entity_view_mode(array &$sandbox = NULL): void {
+  return; // causing updb error in D10 upgrade
   $config_entity_updater = \Drupal::classResolver(ConfigEntityUpdater::class);
 
   $callback = function (EntityViewModeInterface $entity_view_mode) {
