diff --git a/src/LinkitMediaLibraryEditorOpener.php b/src/LinkitMediaLibraryEditorOpener.php
index 88e5a55..3158814 100644
--- a/src/LinkitMediaLibraryEditorOpener.php
+++ b/src/LinkitMediaLibraryEditorOpener.php
@@ -75,7 +75,7 @@ class LinkitMediaLibraryEditorOpener implements MediaLibraryOpenerInterface {
         'data-entity-type' => 'media',
         'data-entity-substitution' => 'media',
         'data-entity-uuid' => $selected_media->uuid(),
-        'href' => '/media/' . $selected_media->id(),
+        'href' => $selected_media->label(),
         'target' => '_blank',
       ],
     ];
