diff --git a/search_api.install b/search_api.install
index 6eae352..42a7184 100644
--- a/search_api.install
+++ b/search_api.install
@@ -104,7 +104,7 @@ function search_api_requirements($phase) {
             'The following search servers are not available: @servers',
             ['@servers' => implode(', ', $unavailable_servers)]
           ),
-          'severity' => REQUIREMENT_ERROR,
+          'severity' => REQUIREMENT_WARNING,
         ],
       ];
     }
