diff --git a/search_api_solr.install b/search_api_solr.install
index b52409a..1bbb232 100644
--- a/search_api_solr.install
+++ b/search_api_solr.install
@@ -217,7 +217,7 @@ function search_api_solr_requirements($phase) {
               ':url' => $server->toUrl('canonical')->toString(),
               '@server' => $server->label(),
             ]),
-            'severity' => REQUIREMENT_ERROR,
+            'severity' => REQUIREMENT_WARNING,
           ];
         }
       }
